apps/recnet-api/src/database/repository/user.repository.ts

```diff
@@ -156,4 +164,13 @@ export default class UserRepository {
         throw new Error("Provider not supported");
     }
   }
+
+  private incensitiveContains(field: string, word: string) {
```

Comment:
```suggestion
  private insensitiveContains(field: string, word: string) {
```
typo

Reply:
And why create this function again?

Reply:
Actually,  this method is not using now. Deleted.